configure.ac

```diff
@@ -192,6 +192,8 @@
     AC_CHECK_FUNCS([strlcpy strlcat])
     CFLAGS=$OCFLAGS
 
+    AC_CHECK_FUNCS([madvise posix_fadvise], [AC_DEFINE([HAVE_ADVISE], [1], [Define if system is willing to listen])])
```

Comment:
Looks like this check is not correct. It defines HAVE_ADVICE 1 even if the checks failed.